dev/AppNotifications/AppNotificationContentBuilder/AppNotificationContentBuilder.idl

```diff
@@ -0,0 +1,138 @@
+﻿import "oaidl.idl";
+import "ocidl.idl";
+
+[contract(AppNotificationsContract, 1)]
+runtimeclass AppNotificationButton
```

Comment:
Which API in the Builder class is using this Button construct? I don't see it being used anywhere?